packages/ui/src/proposals/modals/AddNewProposal/helpers.ts

```diff
@@ -50,6 +50,9 @@ export const defaultProposalValues = {
   updateChannelPayouts: {
     cashoutEnabled: true,
   },
+  updatePalletFrozenStatus: {
+    pallet: 'ProjectToken',
+  },
```

Comment:
I'm not 100% that would fix the failing test but could you try to set a default value for freeze to `false`